src/ui/styles/ContentToggle.scss

```diff
@@ -0,0 +1,24 @@
+.typography__label--helper-text {
+    min-height: 0;
+    margin: 0;
+}
+
+.content-toggle {
+  margin-bottom: 4.5rem;
+
+  label {
+    margin-bottom: 0.375rem;
+  }
+}
+
+.content-toggle__label {
+    display: block;
+}
+
+.content-toggle__toggle-btn {
+  cursor: pointer;
+}
+
+.content-field__input {
```

Comment:
I can see this isn't used here, but likely will be in a later task though. I suspect though that the css here should be specific to the component and not include styles for other parts hence I think it should be removed.